site/docs/install.md

```diff
@@ -9,8 +9,7 @@ See the instructions for installing Bazel on:
 
 *   [Linux](install-ubuntu.md)
 *   [Mac OS X](install-os-x.md)
-*   [Windows (experimental)](install-windows.md)
-*   [CentOS (experimental)](install-ubuntu.md)
+*   [Windows](install-windows.md)
 
 For other platforms, you can try to [compile from source](install-compile-source.md).
```

Comment:
Are there platforms other than Linux, macOS, or Windows? If not, I'll remove this line (we point to compile from source from the other topics)